{
  try {
    listener.testEnded(host);
  }
 catch (  Exception ex) {
    log.error("",ex);
  }
}
